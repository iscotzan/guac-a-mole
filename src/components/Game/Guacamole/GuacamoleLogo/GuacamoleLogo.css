.guacamoleLogo {
  font-family: "Baloo Tamma", cursive;
  /* width: 100%; */
  color: white;
  font-size: 2.5rem;
  letter-spacing: 0.2rem;
  /* position: absolute; */
  text-shadow: -1px -1px 0px blue, 3px 3px 0px blue, 6px 6px 0px darkblue; }
