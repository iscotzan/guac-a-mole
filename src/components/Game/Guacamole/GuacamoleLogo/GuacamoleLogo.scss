$clr-blue: blue;
$clr-white: white;
$clr-blue-900: darkblue;

.guacamoleLogo {
  font-family: "Baloo Tamma", cursive;
  // text-align: center;
  /* width: 100%; */
  // margin: 0px auto;
  color: white;
  font-size: 2.5rem;
  letter-spacing: 0.2rem;
  /* position: absolute; */
  text-shadow: -1px -1px 0px blue, 3px 3px 0px blue, 6px 6px 0px darkblue;
}
//   font-family: 'Baloo Tamma', cursive;
//   text-align: center;
//   width: 100%;
//   margin: 0px auto;
//   color: $clr-white;
//   font-size: 3rem;
//   letter-spacing: 5px;
//   // top: 50%;
//   position: absolute;
//   // margin-top: -40px;

//   text-shadow: -1px -1px 0px $clr-blue,
//     3px 3px 0px $clr-blue,
//     6px 6px 0px $clr-blue-900;
// }
