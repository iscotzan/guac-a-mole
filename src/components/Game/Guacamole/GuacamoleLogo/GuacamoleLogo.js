import * as React from "react";
import "./GuacamoleLogo.css";
const GuacamoleLogo = props => {
  return (
    <h2 className={"guacamoleLogo"}>Guacamole</h2>
  );
};

export default GuacamoleLogo;
