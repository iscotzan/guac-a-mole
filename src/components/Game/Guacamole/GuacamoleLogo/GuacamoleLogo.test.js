import React from "react";
import GuacamoleLogo from "./GuacamoleLogo";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

const wrapper = shallow(<GuacamoleLogo />);
describe("<GuacamoleLogo />", () => {
  it("should be defined", () => {
    expect(wrapper).toBeDefined();
  });

  it("renders without crashing", () => {
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
