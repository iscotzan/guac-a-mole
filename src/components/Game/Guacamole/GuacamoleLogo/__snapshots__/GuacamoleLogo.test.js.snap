// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GuacamoleLogo /> renders without crashing 1`] = `
<h2
  className="guacamoleLogo"
>
  Guacamole
</h2>
`;
